config/routes.rb

```diff
@@ -22,9 +22,10 @@
     get 'anvandare/registrera' => 'devise/registrations#new', as: :new_user_registration
 
     #sessions
-    get 'logga_in' => 'devise/sessions#new', as: :new_user_session
-    post 'logga_in' => 'devise/sessions#create', as: :user_session
-    delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
+    get 'logga-in' => 'devise/sessions#new', as: :new_user_session
+    get 'logga_in', to: redirect('logga-in'), status: 301
+    post 'logga-in' => 'devise/sessions#create', as: :user_session
+    delete 'logga-ut' => 'devise/sessions#destroy', as: :destroy_user_session
```

Comment:
minor inconsistency

"get logga_in" got a redirect but neither of
"post logga-in";
"delete logga-ut"

got one.

My advice? We don't have any API or anything like that. No other people depend on our Urls. Just drop the old Url and be done with it.